docs/www/tls-kubernetes.md

```diff
@@ -0,0 +1,375 @@
+
+This functionality is only available for the Kafka API.
+
+To provide an existing certificate issuer we need to fill in the `issuerRef`.
+
+In the `yaml` configuration file add this section:
+
+```yaml
+    kafkaApi:
+    - port: 9092
+      tls:
+        enabled: true
+        issuerRef:
+          name: <issuer-name>
+          kind: ClusterIssuer
+```
+
+Similarly, to provide a particular certificate (as a secret) as the node certificate we need to fill in the `nodeSecretRef` field:
+
+```yaml
+    kafkaApi:
+    - port: 9092
+      tls:
+        enabled: true
+        nodeSecretRef:
+          name: cluster-tls-node-certificate
+          namespace: given-cert
+```
+
+If the secret is in a namespace different from the namespace of the Redpanda cluster, the operator copies it over.
+
+### Reading/Inspecting the certificates
+
+When enabling TLS, the Redpanda operator generates a root certificate for the Kafka API that is consecutively used to generate a “node” certificate for the Kafka API listener. 
+
+If client authentication (mTLS) is enabled, an additional “client” certificate is generated. Each certificate is stored in a Kubernetes secret resource under the same namespace as the Redpanda cluster. The Certificate name is the same as that of the corresponding secret.
+
+Here’s the list of secrets for the Kafka API:
+
+Root certificate
+* Used to generate the node and client certificates
+* Secret: `<cluster>-kafka-root-certificate`
+
+Node certificate
+* The node certificate is automatically provided to Redpanda by the operator. In particular, the certificate secret is mounted as a volume that is consumed by Redpanda.
+* Secret: `<cluster>-redpanda`
+
+Client certificate (applicable only when client authentication is enabled)
+* The client certificate is available for consumption. Depending on where the client is located, the client certificate secret can be mounted to local pods or be exported out of the Kubernetes cluster, as long as the Admin API is exposed externally.
+* Secret: `<cluster>-user-client`
+* Available for client use
+
+Here’s the table of certificates, the “client” certificates are only needed and generated for client authentication (mTLS).
+
+
+<table>
+<tr>
+<td>Used by/API
+</td>
+<td>Admin API
+</td>
+<td>Kafka API
+</td>
+<td>Pandaproxy API
+</td>
+</tr>
+<tr>
+<td>Node
+</td>
+<td>
+
+`<cluster>-admin-api-node`
+
+</td>
+<td>
+
+`<cluster>-redpanda`
+
+</td>
+<td>
+
+`<cluster>-proxy-api-node`
+
+</td>
+</tr>
+<tr>
+<td>Client
+</td>
+<td>
+
+`<cluster>-admin-api-client`
+
+</td>
+<td>
+
+`<cluster>-user-client`
+
+</td>
+<td>
+
+`<cluster>-proxy-api-client`
+
+</td>
+</tr>
+</table>
+
+Each Certificate resource results in the creation of a secret resource of type `kubernetes.io/tls` that contains three components: `tls.crt`, `tls.key`, and `ca.crt`. 
+
+You can check them by running:
+
+```bash
+kubectl describe secret <cluster>-admin-user-client
+```
+
+The result should look like this: 
```

Comment:
```suggestion
Which will return this: 
```

Reply:
This one I would like to keep because we already have it in other guides. And besides, the results can vary depending on what the user is doing and what names he gave for the things.

Reply:
The problem with this is the word "should."  - https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/should-vs-must
Quote from link - "Use should only to describe an action that's recommended but optional. Don't use should to indicate probability. If you can't make a definitive statement, use might or rephrase."

"Should" should not be used in the docs. That will be in the style guide that I write for us someday. I don't care what you say here, but I do think that word should be removed.